### DISCUSSION TITLE
remove extra line on composed solid

### DISCUSSION THREAD

**Discussion by xyont** [Macad3D/Macad3D#45] [General]
hello,

firstly, i would to say many thanks for such great 3D software.

some problem i found, when the model needed to be use for simulation FEA, commonly it's required to remove extra line/curve for meshing simplification. but sometime these are still needed for some reasons.

is there any possibility  for such purpose? an option to remove or not could be useful.

regards,

**Comment by dipts** [comment]
Macad does not offer such functionality for now. If such a functionality is available in OCCT (you would have to ask there), this can be easily triggered and executed using a script. The script cann insert the newly created TopoDS_Shape as a new body with shape Solid (or replace the shape of the original body).

**Comment by xyont** [comment]
it seems to be right, Saloma have special operation menu for removing extra edge.

hope, Macad3D in next or future versions also have such functionality simplified by option at `Save as` menus.
